NEWS.md

```diff
@@ -1,5 +1,7 @@
 # dials (development version)
 
+* All messages, warnings and errors has been translated to use {cli} package (#311). 
```

Comment:
Added the bullet in this once since I assume it will be the last once to be merged